features/Yves/Context/SprykerInitContext.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Yves\Context;
+
+use Behat\Behat\Context\Context;
+use Spryker\Shared\Library\SystemUnderTest\SystemUnderTestBootstrap;
+use SprykerInitContextTrait;
+
+/**
+ * Inits Spryker test environment for Yves
+ */
+class SprykerInitContext implements Context
```

Comment:
`SprykerInitContext` could be replaced by the behat extension, but works for now